### PR TITLE
Show locals in completion following if( with no space

### PR DESCRIPTION
**Pull request by rchande**
Fixes https://devdiv.visualstudio.com/DevDiv/NET%20Developer%20Experience%20IDE/_queries?id=432231&_a=edit&triage=true"

### CODE REVIEW

**Comment by dpoeschl** [comment]
Developer Community link: https://developercommunity.visualstudio.com/content/problem/52000/local-variables-not-showing-up-in-intellisense-for.html

**Comment by rchande** [comment]
retest windows_release_vs-integration_prtest

File: src/Workspaces/VisualBasic/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.vb [hunk 1]
```diff
@@ -1063,8 +1063,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
             tokenOnLeftOfPosition = tokenOnLeftOfPosition.GetPreviousTokenIfTouchingWord(position)
 
             If tokenOnLeftOfPosition.IsKind(SyntaxKind.OpenParenToken) Then
-                Return tokenOnLeftOfPosition.Parent.IsKind(SyntaxKind.ParenthesizedExpression,
-                                                           SyntaxKind.TupleExpression, SyntaxKind.TupleType)
+                ' If(bar$$
+                ' Exclude parenthesized expressions inside if statements
+                If (tokenOnLeftOfPosition.Parent.IsKind(SyntaxKind.ParenthesizedExpression)) Then
+                    Return Not tokenOnLeftOfPosition.Parent.Parent.IsKind(SyntaxKind.IfStatement)
```
Comment (jasonmalinowski) [inline src/Workspaces/VisualBasic/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.vb]: Why does this change actually work? This is entirely valid (if a bit silly) code:

    If (x, x).Equals((0, 0)) Then

Does this impact that?

Comment (jasonmalinowski) [reply src/Workspaces/VisualBasic/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.vb]: Also, what about other constructs like Do While?

**Pull request closed without merging**